xml/decoders/Zimo_Unified_software_v37_MX618.xml

```diff
@@ -71,7 +72,7 @@
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV153-CV157version28.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV158version32.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV161-CV185Servo.xml"/>
-     <!-- CV7 set to default 37 -->
```

Comment:
Why is this comment removed? The comment is retained in other xml files in this PR. The comment applies to line 76.

Reply:
No idea how this happened. I would only insert line 75 afte line 73.

Sorry for this mistake

Reply:
No problem. Thanks for fixing it.